docs/src/components/landing-page/MainLayoutHeader.vue

```diff
@@ -1,119 +1,249 @@
-      class="primary-toolbar q-pl-lg q-pr-md justify-between items-stretch"
-    >
-      <q-btn v-if="$q.screen.xs" flat @click="$emit('update:modelValue', !modelValue)" round dense icon="menu" color="lp-primary"/>
-      <div v-if="$q.screen.gt.xs || !isSearchFieldActive" class="row justify-center items-center cursor-pointer" @click="$router.push({name: 'home'})">
-        <img v-if="$q.screen.sm" :src="`https://cdn.quasar.dev/logo-v2/svg/logo${!dark? '-dark':''}.svg`" width="48" height="48" alt="Quasar Logo">
-        <img v-else :src="`https://cdn.quasar.dev/logo-v2/svg/logo-horizontal${!dark? '-dark':''}.svg`" width="236" :height="$q.screen.xs? '24':'48'" alt="Quasar Logo">
-        <q-separator v-if="$q.screen.gt.xs" :color="dark? 'black-12':'lp-primary'" vertical class="q-ml-lg" />
-      </div>
+  <transition
+    enter-active-class="animated slideInDown"
+    leave-active-class="animated slideOutUp"
+  >
+    <q-header
+      v-if="scrollOffset < SWAP_HEADER_OFFSET"
+      ref="layoutHeader"
+      :class="dark? 'bg-white text-black-54':'bg-lp-dark text-white-54'"
+      class="font-monserrat lp-header">
+      <q-toolbar
+        :class="{ 'shadow-bottom-small': $q.screen.xs, 'letter-spacing-25': $q.screen.lt.lg, 'letter-spacing-300': $q.screen.gt.md }"
+        class="primary-toolbar q-pl-lg q-pr-md justify-between items-stretch text-size-16"
+      >
+        <q-btn
+          v-if="$q.screen.xs"
+          color="lp-primary"
+          dense
+          flat
+          icon="menu"
+          round
+          @click="$emit('update:modelValue', !modelValue)"/>
+        <div
+          v-if="$q.screen.gt.xs || !isSearchFieldActive"
+          class="row justify-center items-center cursor-pointer"
+          @click="$router.push({name: 'home'})">
+          <img
+            v-if="$q.screen.sm"
+            :src="`https://cdn.quasar.dev/logo-v2/svg/logo${!dark? '-dark':''}.svg`"
+            alt="Quasar Logo"
+            height="48"
+            width="48">
+          <img
+            v-else
+            :height="$q.screen.xs? '24':'48'"
+            :src="`https://cdn.quasar.dev/logo-v2/svg/logo-horizontal${!dark? '-dark':''}.svg`"
+            alt="Quasar Logo"
+            width="236">
+          <q-separator
+            v-if="$q.screen.gt.xs"
+            :color="dark? 'black-12':'lp-primary'"
+            class="q-ml-lg"
+            vertical/>
+        </div>
 
-      <div class="row items-center text-size-16">
-        <div v-if="$q.screen.gt.xs && showNavItems" class="toolbar-menu-items">
-          <q-btn
-              v-for="(navItem, navItemIndex) in navItems.mainNavItems" :key="navItemIndex"
-              :label="navItem.label"
-              :to="computeRouteNav(navItem)"
-              :href="computeRouteNav(navItem, 'href')"
-              :target="navItem.href? '_blank':'_self'"
-              :color="setActivePrimaryNavColor($route.path, navItem.path)"
+        <div class="row items-center">
+          <div
```

Comment:
When search is in "box" mode, you need to add a small margin between the last button and the search box